WORKSPACE

```diff
@@ -70,7 +70,7 @@ grpc_extra_deps()
 git_repository(
      name = "nvriva_common",
      remote = "https://github.com/nvidia-riva/common.git",
-     commit = "1c7da5aed4e4df3a296d2672379c5099a193aaae"
+     commit = "75afa27124dde29a5e892e6817d9841c6cbc49e7"
```

Comment:
This needs to change once common's change is committed?

Reply:
yes